src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,5 @@
+html {
+  font-family: "Roboto", sans-serif;
```

Comment:
Checklist:
2. [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/038ff33a-ce36-4d82-8917-0b6157c07548)